### PR TITLE
Add Model Name to Model Lookup

### PR DESCRIPTION
**Pull request by manstis**
Screenshot from 2020-05-15 09-50-40

### CODE REVIEW

File: front-end-poc/src/Audit/types.ts [hunk 1]
```diff
@@ -21,16 +21,18 @@ export interface IExecutionRouteParams {
   executionType: ExecutionType;
 }
 
+export interface IServiceIdentifier {
```
Comment (manstis) [inline front-end-poc/src/Audit/types.ts:24]: I did this in preparation for using `version` more later on... not strictly necessary here but will be useful later.

File: front-end-poc/src/Audit/AuditOverview/AuditOverview.tsx [hunk 2]
```diff
@@ -118,7 +118,8 @@ const AuditOverview = () => {
           // temporary solution: for demo purposes we display the first 3 executions here
           if (response.data.total > 0 && latestSearches.length === 0) {
             let searches = [];
-            for (let i = 0; i < 3; i++) {
+            let maxSearches = Math.min(3, response.data.headers.length);
```
Comment (manstis) [inline front-end-poc/src/Audit/AuditOverview/AuditOverview.tsx:121]: Fixed this too... I was wondering why my "Last opened.." was always the `SkeletonInlineStripe` :-)

**Comment by manstis** [comment]
Merging as I know you're busy @kelvah.

File: front-end-poc/src/Audit/AuditOverview/AuditOverview.tsx (hunk 2 above)
Comment (kelvah) [reply front-end-poc/src/Audit/AuditOverview/AuditOverview.tsx:121]: Oops! :facepalm:

**Comment by kelvah** [comment]
Thank you @manstis!

**Pull request merged**